src/lib/cloud/social/provider.ts

```diff
@@ -239,6 +239,8 @@ export class CloudSocialProvider {
           description: banner,
           version: connection.getVersion()
         };
+        // Cloud server must be online if it is responding with it's banner.
```

Comment:
Regarding the comment: I think a better measure is whether the server responded with a **ping** - which is the case once `connect`, above, has resolved.